virtualizarr/tests/test_backend.py

```diff
@@ -189,6 +189,7 @@ def test_var_attr_coords(self, netcdf4_file_with_2d_coords):
             + expected_2d_coords
             + expected_1d_non_dimension_coords
             + expected_scalar_coords
+            + ["xi_rho", "eta_rho"]
```

Comment:
This one I have very little context on. @sharkinsspatial's backend has them and the kerchunk-based one does not. Before this PR the non-kerchunk backend was not being exercised by this test.

Reply:
That's probably just an oversight. This is exercising behaviour to do with determining with variables are coordinate variables, which happens in a layer that occurs for both backends anyway.

Reply:
I now think this is more of #401 so I left it failing.